test/unit/test_account.py

```diff
@@ -607,6 +607,15 @@ def test_enable_account_by_env_input_instance(self):
         self.assertTrue(service._account)
         self.assertEqual(service._account.instance, instance)
 
+    def test_active_account_with_saved_instance(self):
+        """Test active_account with a saved instance."""
+        name = "foo"
+        token = uuid.uuid4().hex
+        instance = "test-hub/test-group/test-project"
+        with temporary_account_config_file(name=name, token=token, instance=instance):
+            service = FakeProvider(name=name)
```

Comment:
Here we are testing FakeProvider whereas we should be actually testing IBMProvider.